2021-10-13-github-templates.md

```diff
@@ -0,0 +1,133 @@
+# OmbuLabs Standard Github Templates
+
+## Problem:
+
+New and junior contributors might be unsure about the information we require to
+efficiently review a PR. Similarly, it might be unclear what information is required when
+a new BUG or FEATURE is logged as an issue in Github.
+
+We have existing PR and ISSUE templates but they are inconsistent and might ask for
+unnecessary information from the contributor.
+
+The goal is to simplify  and streamline the steps required to make a contribution or log
+an issue.
+
+## Proposal:
+
+Come up with a consistent set of Github templates. The templates should include prompts
+only the ESSENTIAL information we require. We want to make sure that we don't ask for more
+information than we require - as this will discourages contributions.
+
+## Templates:
+
+### Pull Requests
+
+```
+    <!--- Before we start... -->
+    <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+    <!--- Did you update any relevant documentation -->
+    <!--- Did you add tests to cover new changes -->
+
+    ## Description
+    <!--- Describe your changes in detail -->
+    <!--- Please read the README before submitting pull requests for this project. -->
+
+    ## Motivation and Context
+    <!--- Why is this change required? What problem does it solve? -->
+    <!--- If it fixes an open issue, please link to the issue here. -->
+
+    ## How Has This Been Tested?
+    <!--- Include any relevant details about your testing environment -->
+
+    ## Screenshots (if appropriate):
+
+    **I will abide by the [code of conduct](<LINK TO CODE OF CONDUCT HERE>)**
+```
+
+
+### Issues: Bug Reports
+
+```
+    <!--- Before we start... -->
+    <!--- Did you check the documentation for answers? -->
+    <!--- Did you sure that this issue has not already been filed? -->
+
+    ## Expected Behavior
+    <!--- Tell us what should happen -->
+
+    ## Actual Behavior
+    <!--- Tell us what happens instead -->
+
+    ## Possible Fix
+    <!--- Optionally suggest a fix or reason for the bug -->
+
+    ## Steps to Reproduce
+    <!--- Provide a link to a live example, or an unambiguous set of steps to -->
+    <!--- reproduce this bug. Include code to reproduce, if relevant -->
+    1.
+    2.
+    3.
+    4.
+
+    ## Your Environment
```

Comment:
How about condensing these last 3 items in one? something like `Extra information` `Include any relevant details about your environment (branch, browser, OS) and the bug (screenshots, logs, etc)`

Reply:
Good suggestion ✅